README.md

```diff
@@ -1,6 +1,10 @@
 Simple usage
 ============
 
+## First Things First
```

Comment:
Hey @m4dc4p ,

Thanks a lot for this PR! Lowering the entry barrier for newcomers is indeed a very good goal.

However, I'm not convinced that this section transports the information as good as it could. I'd like to propose to replace it by a short introduction at the top of this file that covers:
- This nix environment is meant for GHC development.
- You need a cloned GHC repo.
- Link to https://ghc.dev

So, basically what you already have, but reverb `This shell is intended to run in the context of the GHC repo` and move it to the top.

Would be great if you could find some time to do this. Otherwise, I can.